javascript_es6/1.a.browser-echobot/src/app.js

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { BlobStorage, CosmosDbStorage } from 'botbuilder-azure';
+import { BotStateSet, ConversationState, MemoryStorage, UserState, ActivityTypes } from 'botbuilder-core';
+import 'botframework-webchat/botchat.css';
+import { App } from 'botframework-webchat/built/App';
+import './css/app.css';
+import { WebChatAdapter } from './webChatAdapter';
+
+// Instantiate MemoryStorage for use with the ConversationState middleware.
+// Below are examples of different memory storage offerings that use Azure Blob and Cosmos DB storage.
+const memory = new MemoryStorage();
+
+// To use Azure Blob Storage to store memory, you can the BlobStorage class from `botbuilder-azure`.
+// When using BlobStorage either a host string or a `Host` interface must be provided for the host parameter.
+// const blobStorageHost = {
+//     primaryHost: '',
+//     secondaryHost: ''
+// }
+// const memory = new BlobStorage({
```

Comment:
>BlobStorage [](start = 22, length = 11)

Does the Botbuilder-azure dependencies make sense for a browser bot? I'm pretty sure they (in turn) have Node dependenceis are are not suitable for browser bots.

Reply:
[Taken out](https://github.com/Microsoft/BotBuilder-Samples/pull/317#discussion_r216100594)